ci-operator/config/openshift/ceph-csi/openshift-ceph-csi-release-4.7.yaml

```diff
@@ -0,0 +1,21 @@
+build_root:
+  project_image:
+    dockerfile_path: scripts/Dockerfile.test
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: static-checks
+  commands: |
+    ln -s $PWD ../../ceph/ceph-csi
```

Comment:
The `../../ceph` directory probably does not exist, you will need to create it before creating the symlink. I do not see an error that creating the symlink failed, so I am not 100% sure.

What you could do is:
```
  commands: |
    mkdir ../../ceph && ln -s ../openshift/ceph-csi ../../ceph/ceph-csi
    GOCACHE=/tmp XDG_CACHE_HOME=/tmp PYLINTHOME=/tmp make cephcsi test
```

Reply:
No ceph directory exist, initially I did mkdir which failed saying that it exist and the reason it exists is because we create it as a WORKDIR while building root.

But looks like symlink wont work for GO, I faced that issue and tried copying the whole source code instead of creating a symlink. After doing that import error is not there but other errors are coming see the changes in master yaml.
I tried copy in master yaml and symlink in release-4.7 yaml.

Reply:
Yes, I just spend a minute going through the `gosec` failures. An other approach you may want to try:

```
  commands: |
    mkdir ../../ceph && ln -s ../openshift/ceph-csi ../../ceph/ceph-csi
    cd ../../ceph/ceph-csi
    GOCACHE=/tmp XDG_CACHE_HOME=/tmp PYLINTHOME=/tmp make cephcsi test
```

Run the commands from within the `/go/src/github.com/ceph/ceph-csi` (symlinked) directory. This might trick Golang enough to accept the `internal/` import path.